test/verify/check-shell-keys

```diff
@@ -39,7 +39,7 @@ class TestKeys(MachineCase):
 
         m.start_cockpit()
 
-        def login(user, password, user_name):
+        def login(user, password):
```

Comment:
That commit message has "more shell keys", which probably refers to the other unused variables. Not a biggie, just nice to clean up if you have to force-push again for some other reason.

Reply:
Oops.  That's from squashing the fixup to the last round of test failures.  I'd prefer not to merge that, so I'll fix the message and try again.  I'm not in a hurry.